### PR TITLE
Dsg-2226 initial new application select options

### PR DESCRIPTION
**Pull request by g8shen**
# Basics

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [x] Feature
- [ ] Styling and layout changes
- [ ] Code style changes (formatting, local variables)
- [ ] Refactoring (no functional changes)
- [ ] Build or CI related changes
- [ ] Tests
- [ ] Other... Please describe:

## What changes does this PR introduce?

New application now has a "button dropdown" and choose from a hardcoded list of applications for now.

New "button dropdown" component.
## Screenshot of the affected changes

Before:
Screenshot 2023-11-08 at 12 25 23 PM

After:

https://github.com/Nuvalence/dsgov-web/assets/39806680/fae7263a-d763-4ce9-b273-c5169ba3c31d

## Checklist

- [ ] I have updated the documentation accordingly
- [ ] I have confirmed all acceptance criteria defined by the story have been met
- [ ] I have confirmed that Sonar Qube findings for new code are reviewed and fixed, or will be whitelisted

### CODE REVIEW

File: libs/shared/ui/nuverial/src/lib/components/selector-button-dropdown/selector-button-dropdown.component.html [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    </div>
+  </button>
+</div>
+
+<mat-menu #menu="matMenu">
+  <div class="menuItem">
+    <button
+      class="custom-menu-item"
+      *ngFor="let item of menuItems; trackBy: trackByFn"
+      [ngStyle]="{'background-color': item.selected ? ' var(--theme-color-m3-neutral-90)' : ''}"
```
Comment (g8shen) [inline libs/shared/ui/nuverial/src/lib/components/selector-button-dropdown/selector-button-dropdown.component.html:15]: I tried to NgClass here or class.selected but it did not apply the classes. I also tried with more specifity and !important

File: libs/public/feature/dashboard/src/lib/components/dashboard/dashboard.component.ts [hunk 2]
```diff
@@ -27,14 +42,28 @@ export class DashboardComponent {
     private readonly _profileService: PublicFeatureProfileService,
     private readonly _dashboardService: DashboardService,
   ) {}
+  public selectOption: INuverialSelectOption[] = [
```
Comment (davidyoung77) [inline libs/public/feature/dashboard/src/lib/components/dashboard/dashboard.component.ts:45]: Is the goal of this story to add additional hard coding?

File: libs/shared/ui/nuverial/src/lib/components/selector-button-dropdown/selector-button-dropdown.component.html (hunk 1 above)
Comment (davidyoung77) [reply libs/shared/ui/nuverial/src/lib/components/selector-button-dropdown/selector-button-dropdown.component.html:15]: You are right this should be ngclass

File: libs/public/feature/dashboard/src/lib/components/dashboard/dashboard.component.ts (hunk 2 above)
Comment (g8shen) [reply libs/public/feature/dashboard/src/lib/components/dashboard/dashboard.component.ts:45]: The ticket specifies hardcoded values for the moment: [](https://dol-ewf.atlassian.net/browse/DSG-2226) I assume it'll eventually need to come from an API

**Review by davidyoung77 — APPROVED** [review]
Looks good thanks

**Pull request merged**